<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>addForm</class>
 <widget class="QWidget" name="addForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>325</width>
    <height>232</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Добавление</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="2" column="0">
    <widget class="QPushButton" name="addButton">
     <property name="text">
      <string>Добавить</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QPushButton" name="exitButton">
     <property name="text">
      <string>Назад</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="2">
    <layout class="QFormLayout" name="formLayout">
     <item row="0" column="0">
      <widget class="QLabel" name="num">
       <property name="text">
        <string>Номер</string>
       </property>
      </widget>
     </item>
     <item row="0" column="1">
      <widget class="QLineEdit" name="numEdit"/>
     </item>
     <item row="1" column="0">
      <widget class="QLabel" name="model">
       <property name="text">
        <string>Модель</string>
       </property>
      </widget>
     </item>
     <item row="1" column="1">
      <widget class="QLineEdit" name="modelEdit"/>
     </item>
     <item row="2" column="0">
      <widget class="QLabel" name="year">
       <property name="text">
        <string>Год производства</string>
       </property>
      </widget>
     </item>
     <item row="2" column="1">
      <widget class="QLineEdit" name="yearEdit"/>
     </item>
     <item row="3" column="0">
      <widget class="QLabel" name="life_time">
       <property name="text">
        <string>Срок службы</string>
       </property>
      </widget>
     </item>
     <item row="3" column="1">
      <widget class="QLineEdit" name="life_timeEdit"/>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="QLabel" name="inf">
     <property name="autoFillBackground">
      <bool>true</bool>
     </property>
     <property name="text">
      <string/>
     </property>
     <property name="scaledContents">
      <bool>false</bool>
     </property>
     <property name="wordWrap">
      <bool>false</bool>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
